srcpkgs/sxiv/template

```diff
@@ -15,6 +15,10 @@ pre_build() {
 	[ -e ${FILESDIR}/config.h ] && cp ${FILESDIR}/config.h config.h ||:
 }
 
+post_install() {
+  vinstall sxiv.desktop 644 ${DESTDIR}/usr/share/applications
```

Comment:
vinstall doesn't need ${DESTDIR} it always perform against it

usr/share/applications is correct

Reply:
I was unsure of that, fixed!